cmd/dummy/main.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"errors"
+	"log"
+	"net"
+	"os"
+	"sync"
+	"time"
+
+	"google.golang.org/grpc"
+
+	grpcproxy "github.com/rollkit/go-execution/proxy/grpc"
+	"github.com/rollkit/go-execution/test"
+	pb "github.com/rollkit/go-execution/types/pb/execution"
+)
+
+const bufSize = 1024 * 1024
```

Comment:
any reason behind these numbers we should capture?

Reply:
Sorry, mostly copy-paste ;)